src/Microsoft.DotNet.Interactive.Documents/InteractiveDocument.cs

```diff
@@ -134,16 +160,19 @@ internal void NormalizeElementKernelNames(KernelInfoCollection kernelInfos)
         return null;
     }
 
-    public static async Task<InteractiveDocument> LoadInteractiveDocumentAsync(
-    FileInfo file,
-    KernelInfoCollection kernelInfos)
+    public static async Task<InteractiveDocument> LoadAsync(
```

Comment:
If LoadAsync expands / resolves the imports always could this cause a notebook parsing to inline the imported notebooks as we open a ipynb / dib in vscode?

Reply:
It doesn't resolve them or even try to identify them until someone calls `GetImportsAsync`.